src/navigation/index.tsx

```diff
@@ -10,6 +10,20 @@ import { Content } from "./Content";
 import { Providers } from "../context/composeProviders";
 import { DrawerContextProvider } from "../context/drawer";
 import { CampaignConfigsStoreContextProvider } from "../context/campaignConfigsStore";
+import * as Localization from "expo-localization";
+import zh from "../common/i18n/translations/zh.json";
+import en from "../common/i18n/translations/en.json";
+import i18n from "i18n-js";
+
+i18n.fallbacks = true;
+i18n.locale = Localization.locale;
+i18n.translations = {
+  zh,
+  en
+};
+i18n.missingTranslation = function () {
+  return undefined;
+};
```

Comment:
Why do we need this? When we already have `i18n.fallbacks`?

Reply:
Good point, will remove this!

Reply:
`i18n.missingTranslation` allows us to handle situations where some modals do not have a `body` or `secondaryActionText` by simply returning `undefined` so nothing is shown.

Let me know if this is okay.

Reply:
But what if we made a mistake and the translation is missing from chinese but is available in English?

Reply:
Just tested how the fallback will work:

1. If the set locale language does not exist, the default locale language will be displayed.
2. If the default locale language translation does not exist, `undefined` will be returned.
3. `showErrorAlert` dialog content will be the error's message.